_layouts/index.html

```diff
@@ -91,12 +91,20 @@
       </div>
 
       <div class="col-md-4">
+        <br>
+        <h2 style="text-align: center;">
+          <a href="https://www.teepublic.com/user/bids">Checkout BIDS Merch</a>
+          </h2>
+          <p style="text-align: center;">
+          <div>All proceeds from sales are donated to the Wikimedia Foundation</div>
```

Comment:
They are the same to me, @christinerogers feel free to suggest and commit whatever :)

Reply:
@bendhouseart There is an unfortunate difference, since we don't manufacture the merch, we only get paid profits and have no control over all proceeds.

From https://www.zippia.com/advice/proceeds-vs-profits/:

> Proceeds are the entire amount received in income. Profits are only the amount received over the cost of buying the goods sold.

So @christinerogers is quite right here!